### DISCUSSION TITLE
Allow cURL installation of specific Meilisearch releases

### DISCUSSION THREAD

**Discussion by JasonObeid** [meilisearch/product#659] [Feedback & Feature Proposal]
Currently the [instructions](https://www.meilisearch.com/docs/learn/getting_started/installation) for installing Meilisearch via cURL only supports installing the latest release. 

It would be great if we could update the [install script](https://github.com/meilisearch/meilisearch/blob/v1.1.1/download-latest.sh) to support installing a specific release version with something like an environment variable or a flag.

This will help to ensure we know which version we are running locally, which will give us confidence that we are running the same version of Meilisearch locally and in production.

Happy to help with contributions. Thank you!

**Comment by schlenks** [comment]
This improvement to the install script would be really helpful when a project wants to pin their version of meilisearch because it's stable within their environment or they want to ensure all internal environments are guaranteed on the same version.
